app/views/organisations/show.html.erb

```diff
@@ -12,6 +12,10 @@
            schema: :organisation,
            content_item: @organisation.content_item.content_item_data %>
 
+<% @organisation.content_extensions&.each do |content_extension| %>
+  <%= render partial: "content_extension_#{content_extension["document_type"]}", locals: { content_extension: } %>
+<% end %>
+
```

Comment:
If there are multiple "hub" like pages that behave a bit like parts, it'd probably be better to render this new type of content in frontend, that already handles parts.

Reply:
I'm assuming we're adorning the existing organisation pages with some extra components here. Appreciate that might not be the approach we end up taking though.